external/harvestPackages/harvestPackages.props

```diff
@@ -116,7 +116,7 @@
   <Version>4.5.0</Version>
 </PackageReference>
 <PackageReference Include="System.Security.Principal.Windows">
-  <Version>4.5.0</Version>
+  <Version>4.5.1</Version>
```

Comment:
I guess you just ran the updateHarvest target?

Reply:
Yep